katabatic/models/GANBLR++/GANBLR++.py

```diff
@@ -0,0 +1,82 @@
+#install ganblr 
+!pip install ganblr
```

Comment:
include in seperate requirements file

Reply:
this is the weeks ago base "template" for the model, the newer functional mode need not the requirement as the ganblr is implemented directly